multitool/private/multitool.bzl

```diff
@@ -233,6 +233,16 @@ def hub(name, lockfiles):
     _multitool_hub(name = name, lockfiles = lockfiles)
 
 def multitool(name, lockfile):
-    "(non-bzlmod) Create a multitool hub and register its toolchains."
-    hub(name, [lockfile])
+    """(non-bzlmod) Create a multitool hub and register its toolchains.
+
+    Args:
+        name: resulting "hub" repo name to load tools from
+        lockfile: a label or list of labels for lockfiles, see /lockfile.schema.json
```

Comment:
I think it might be less confusing to have `multitool` take `lockfile` and `lockfiles` as arguments and to fail if more than one is set. Depending on which one is set we can do the right thing.

Also wouldn't be terribly opposed to just changing it to take a list always, it's a pretty minor breaking change.

Reply:
sure, can switch to that, PTAL